data/i18n/en/en.toml

```diff
@@ -1,12 +1,7 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
 
-# Avoid using conjunction_1.
-# Must match the context in layouts/shortcodes/release-data.html
-# Appears on https://kubernetes.io/releases/
-# For example the "and" in "Complete 1.25 Schedule and Changelog"
-[conjunction_1]
-other = "and"
+# For "and", see [conjunction_1]
```

Comment:
Maybe it's unnecessary, how do you think?

Reply:
I think it is helpful. Users will most likely search alphabetically by the string value, "and", not by the string name, "conjunction_1".